### DISCUSSION TITLE
Configurable CLI defaults

### DISCUSSION THREAD

**Discussion by ardasugur** [abpframework/abp#9531] [Ideas] [answered]
As I try and create ABP apps with cli, most of times use same parameters. (like angular ui and postgres db) 
Maybe you can create a default setting configuration.( like abp config defaults set ui angular) 
I would make a pr to do it if cli was open source, (I couldn't find the source repo.)

So I want to start a discussion here.

**Comment by maliming** [comment, marked as answer]
hi

Here is the source code of CLI.

https://github.com/abpframework/abp/tree/dev/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli
https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/NewCommand.cs

**Reply by ardasugur** [reply]
Oh, my ignorance I guess.
May I create a pr for this, I think I can implement this.
(with a  run control file on user home folder.)

**Reply by maliming** [reply]
The function is complex and not commonly used. You can create a cmd script to call the `abp new` command to simplify it.

**Reply by ardasugur** [reply]
Yes It's common way to solve problems like this. I created this discussion to see if there are people like me who would want this kind of configuration. 
Thank you for your time and suggestion :)
